runtimes/python-3.10/src/server.py

```diff
@@ -69,7 +69,7 @@ def error(self, message):
         else:
             self.errors.append(str(message))
 
-HTTP_METHODS = ['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'CONNECT', 'OPTIONS', 'TRACE', 'PATCH']
+HTTP_METHODS = ['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'OPTIONS', 'TRACE', 'PATCH']
```

Comment:
why are we removing this ?

Reply:
If I recall correctly, it was to keep consistent methods across runtimes. Not all of them support `CONNECT`.